tests/ui/test_capacity_breakdown_ui.py

```diff
@@ -43,8 +45,10 @@ class TestCapacityBreakdownUI(ManageTest):
             ),
         ],
     )
+    @ui
     @tier2
     @bugzilla("1832297")
+    @skipif_ocs_version("!=4.8")
```

Comment:
Is the capacity breakdown in console different in other versions ?

Reply:
yes